frontend/src/scenes/billing/PlanComparison.tsx

```diff
@@ -278,7 +287,15 @@ export const PlanComparison = ({
                             )}
                         >
                             <Tooltip title={feature.description}>
-                                <span>{feature.name}</span>
+                                <div
+                                    className={
+                                        comparisonModalHighlightedFeatureKey === feature.key
```

Comment:
I wonder if it'd be better to use a light version of the orange we use for the "subscribe" buttons. It might also make more sense to highlight the values for the this feature in the columns for the higher plans rather than the feature name.

It would be pretty neat if the header section of the table, with the plan names and subscribe buttons, was sticky.

Reply:
changed button color, not going to do sticky header for now